### PR TITLE
adding links to all blog posts in the home page #48

### PR DESCRIPTION
**Pull request by mmavroudi**

### CODE REVIEW

File: projects/templates/home.html [hunk 1]
```diff
@@ -69,9 +69,9 @@ <h2>9 Half-up/half-down Hairstyles for Long and Medium Hair</h2>
         <div class="row g-5">
           <div class="col-lg-4">
             <div class="post-entry-1 lg">
-              <a href="single-post.html"><img src="{{ MEDIA_URL }}{{ rand_post.image }}" alt="" class="img-fluid"></a>
+              <a href="{{ rand_post.slug }}"><img src="{{ MEDIA_URL }}{{ rand_post.image }}" alt="" class="img-fluid"></a>
```
Comment (kkampardi) [inline projects/templates/home.html]: Always use a  [url tag](https://docs.djangoproject.com/en/4.1/ref/templates/builtins/#url)  in href in django templates. 
Need to update all the instances that use slug in the href in this file
**To use slug in the href tag is not a good practice, can cause issues in the future.**

**Review by kkampardi — CHANGES_REQUESTED** [review]
It is functional but need to update the home urls

File: projects/templates/home.html (hunk 1 above)
Comment (kkampardi) [reply projects/templates/home.html]: @mmavroudi

**Comment by mmavroudi** [comment]
I don't know if this is the right way of using url tags. It was a bit difficult to understand how I would take the specific url with the specific slug each time and I came across with this solution in stackoverflow. Is it ok?

**Review by kkampardi — APPROVED** [review]

**Pull request merged**